manage-data/lifecycle/data-stream.md

```diff
@@ -22,6 +22,15 @@ To achieve that, it supports:
 
 A data stream lifecycle also supports downsampling the data stream backing indices. See [the downsampling example](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-data-lifecycle) for more details.
 
+## Availability
```

Comment:
I think the heading should be more specific, so maybe like `## ILM availability`

Also, the `## Actions` heading should maybe be something like `## Index lifecycle actions`?

Reply:
I've changed the titles as suggested. Similarly `Data stream lifecycle availability` for the other page.